test/azul_test_case.py

```diff
@@ -185,7 +190,7 @@ def _mock_catalogs(cls):
         except AttributeError:
             pass
         # Patch the catalog property to use a single fake test catalog.
-        catalogs = cls.catalog_config
+        catalogs = cls.catalog_spec
```

Comment:
Do the `json.dumps` here so you don't have to repeat it.

Reply:
Also assert here that `cls.catalog` is the name of the catalog in `cls.catalog_spec`